rskj-core/src/main/java/co/rsk/peg/ReleaseTransactionBuilder.java

```diff
@@ -39,6 +39,10 @@
  * @author Ariel Mendelzon
  */
 public class ReleaseTransactionBuilder {
+
+    public static final int BRIDGE_BTC_TX_VERSION_2 = 2;
+    public static final int BRIDGE_BTC_TX_LEGACY_VERSION = 1;
```

Comment:
```suggestion
    public static final int BTC_TX_VERSION_2 = 2;
    public static final int BTC_TX_LEGACY_VERSION = 1;
```
These Values Are Related To Bitcoin Transactions, Not Necessarily Related To The Bridge.

Reply:
It's kind of related to the bridge. These are the version used by the bridge for the bridge btc transactions(bitcoin txs created by the bridge).  These values are in `bridgeSupport` where context matters, for example: 

![image](https://github.com/rsksmart/rskj/assets/94651730/582db6cd-4a38-4da6-af8d-b6041e244a07)

Reply:
Its Actually a Feature For Creating BTC Transaction, See: https://developer.bitcoin.org/reference/transactions.html#raw-transaction-format